lib/danger/danger_core/plugins/dangerfile_github_plugin.rb

```diff
@@ -92,6 +92,14 @@ def self.instance_name
       "github"
     end
 
+    # @!group PR Review
+    # Current PR Review object.
+    # @return [GitHubReview]
+    #
```

Comment:
This will need considerable documentation before we ship ( as this is what an end user would see on the website ) - I'm happy to do this once you're ready though 👍